Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -608,3 +607,24 @@ void HexEditorWidget::drop_event(GUI::DropEvent& event)
         open_file(response.value().filename(), response.value().release_stream());
     }
 }
+
+DeprecatedString HexEditorWidget::make_display_string(DeprecatedString string)
```

Comment:
i believe that this function could even land in the `String`/`StringView` etc. under the name of `as_printable`/`to_printable` or similar… 🤔

Reply:
I like the idea, though I'm not really confident enough to feel the change is warranted there

Would you recommend taking any actions in that direction? If so, which ones? Maybe you're just floating an idea out there...? 🙂

Reply:
letʼs ignore that then! itʼll definitely find its way to a proper place if/where needed! 👍